### ISSUE TITLE
Editing of File contents not possible when page name is "files"

### ISSUE THREAD

**Issue by tobimori** [k-next/kirby#5273]
## Description
When the page name is `files`, and you're trying to edit a files' contents (e.g. an Alt text field) the `lock` panel request will fail and forbid any editing.

```
{
    "status": "error",
    "message": "Invalid model type: .",
    "code": 400,
    "exception": "Kirby\\Exception\\InvalidArgumentException",
    "key": "error.invalidArgument",
    "file": "../kirby/src/Cms/Find.php",
    "line": 126,
    "details": [],
    "route": "(.*)/lock"
}
```

**Expected behavior**  
File contents editing world work normally as when the page name isn't `files`.

**Screenshots**  
![CleanShot 2023-06-13 at 00 07 52@2x](https://github.com/getkirby/kirby/assets/29142128/d80dedbf-b8e2-4aaf-b85a-9a71eb4503f3)

## To reproduce
1. Create a page with the slug `files`.
2. Upload a file with custom blueprint that has fields.
3. Open the file and try to edit a field

## Your setup
**Kirby Version**  4.0.0-alpha.2 / 3.9.4

**Comment by distantnative** [comment]
This is new with v4?

**Comment by tobimori** [comment]
> This is new with v4?

I have another site running on 3.9.4 where it works just fine.

**Comment by tobimori** [comment]
> This is new with v4?

You're right, it's also happening with 3.9.4, I just haven't noticed it yet because the `lock` calls where less frequent on another site.

**Comment by lukasbestle** [comment]
Looks like the wrong API route matches.

**Comment by distantnative** [comment]
@lukasbestle I think we only could solve this when moving to Panel urls based on UUIDs

**Comment by lukasbestle** [comment]
Would be good to know which exact route causes this, so far it's only an assumption from me reading the issue. Because we separate page levels with `+` in API URLs, I feel like there should be a way to differentiate between "page called `files`" and "files of whatever page".

**Comment by bastianallgeier** [comment]
@distantnative as soon as we move to UUIDs for panel URLs, we can no longer switch them off.

**Comment by distantnative** [comment]
@bastianallgeier true

@lukasbestle so it's a call to the API such as `http://sandbox.test/api/pages/files/files/team.jpg/lock`, I would think all 5 lock API endpoints are affected as they all do pretty much the same with the path.
